### PR TITLE
dcos-monitoring: Update docs for 0.3.0

### PR DESCRIPTION
**Pull request by jieyu**
## Description

Documentation of DC/OS monitoring service v0.3.0 release.

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/beta-dcos-monitoring/0.3.0/operations/grafana/grafana-ui/index.md [hunk 1]
```diff
@@ -3,29 +3,32 @@ layout: layout.pug
...
+#include /services/include/beta-software-warning.tmpl
 
 # Access Grafana UI
 
 ## Access through Admin Router
 
 You should be able to access Grafana UI through Admin Router using "Services" routes by default (unless `grafana.admin_router_proxy` is set to `false`).
 
-Assume the service name is `dcos-monitoring` (default), you can access the Grafana UI using the following URL:
+Assume the service name is `{{ model.serviceeName }}` (default), you can access the Grafana UI using the following URL:
```
Comment (jieyu) [inline pages/services/beta-dcos-monitoring/0.3.0/operations/grafana/grafana-ui/index.md]: typo

**Comment by SarahStegall** [comment]
I have made the following changes:

1. Renamed the directory to /services/beta-dcos-monitoring/ to conform to Mesosphere naming standard.
2.  Created Mustache variable file data.yml, with appropriate variables. Edited versions 0.2.1 and 0.3.0 to add these variables in conformance with Mesosphere standards.
3. Re-organized 0.3.0 files to reflect Mesosphere standards; please note that NO material was deleted, but may have been moved to another location. All material in the original submission has been retained.
4. As part of the re-organization, created an Overview and Getting-Started page to conform to Mesosphere standards.

Please let me know if you have any questions.

File: pages/services/beta-dcos-monitoring/0.3.0/operations/grafana/grafana-ui/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/services/beta-dcos-monitoring/0.3.0/operations/grafana/grafana-ui/index.md]: Fixed. Thanks for finding it.

**Comment by ck4adventure** [comment]
Is this still a relevant PR?? Can it be closed? @SarahStegall

**Comment by SarahStegall** [comment]
Out of date.

**Pull request closed without merging**